### ISSUE TITLE
Could not find a server matching the region en

### ISSUE THREAD

**Issue by BannerBomb** [WillTDA/Discord.js-Akinator#46]
```js
const akinator = require('discord.js-akinator');
await akinator(message, {
	language: 'en',
	childMode: false,
	gameType: 'character',
	useButtons: true,
	translationCaching: {
		enabled: false
	}
});
```

results in the error

```console
Error: Could not find a server matching the region en
    at Akinator.start (/opt/stratobotelite/node_modules/aki-api/dist/src/Akinator.js:49:19)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async module.exports (/opt/stratobotelite/node_modules/discord.js-akinator/src/index.js:118:9)
    at async module.exports.run (/opt/stratobotelite/src/bot/commands/Games/Singleplayer/akinator.js:28:3)
    at async module.exports.runCommand (/opt/stratobotelite/src/bot/monitors/commandHandler.js:75:23)
    at async module.exports._run (/opt/stratobotelite/src/frameworks/klasa/lib/structures/Monitor.js:99:4)
```

**Comment by BannerBomb** [comment]
https://github.com/jgoralcz/aki-api/issues/155

**Comment by WillTDA** [comment]
Hello @BannerBomb, thanks for the issue report. I am aware of the state of the Akinator API as of present, and I will be sure to publish a new version of this package assuming the developer of the depending package provides a fix. 👍

**Comment by WillTDA** [comment]
Thank you! Looking into it now.

**Comment by WillTDA** [comment]
@BannerBomb Fixed in **v4.1.4**, which has now been made available for installation! Please update the package on your system.
